### PR TITLE
Enhancement/New Sound Effects

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

File: Assets/_PartyParkade/Script/Game/Destructibles/Destructible.cs [hunk 1]
```diff
@@ -23,14 +23,21 @@ public abstract class Destructible: MonoBehaviour
...
+        [PropertyOrder(100)]
+        [Title("Audio")] 
+        [SerializeField] private PerigonSoundEffect m_collisionSound = new PerigonSoundEffect();
+        [SerializeField] private float m_activatedSFXVelocity = 5f;
+    
         private Rigidbody m_rigidbody = null;
         private Vector3 m_originalMeshLocalPosition = Vector3.zero;
         private Quaternion m_originalMeshRotation = Quaternion.identity;
 
+        public AudioManager AudioManagerInstance => AudioManager.Instance;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/Game/Destructibles/Destructible.cs:35]: Does this have to be public?? or will protected be okay?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/Destructibles/Destructible.cs:35]: protected is fine

**Review by Gursaanj — APPROVED** [review]
Looks good, I will either get the new SFX in this PR or in another one

**Comment by m4rtint** [comment]
> Looks good, I will either get the new SFX in this PR or in another one

Yeah do it in next PR

**Review by mikeslgoh — APPROVED** [review]
I think for playtesting we should play around with the volume because I still don't like the sound of the wall collision

**Comment by mikeslgoh** [comment]
[SFX for destructible objects (2.0)](https://trello.com/c/kl3UKs5n/753-sfx-for-destructible-objects-20)

**Pull request merged**